code/espurna/sensor.ino

```diff
@@ -593,13 +654,20 @@ void _sensorLoad() {
 
     #if PZEM004T_SUPPORT
     {
-        PZEM004TSensor * sensor = new PZEM004TSensor();
+        PZEM004TSensor * sensor = pzem004t_sensor = new PZEM004TSensor();
```

Comment:
For previous comment, I meant calling hypothetical `pzem004InitCommands(sensor)` here and moving it to sensor header file.

Reply:
I see, but all sensors are instantiated here, so changing this to the sensor itself I believe is not the best solution.

Reply:
Yes, I have not made my mind on how this will work in the future so there is no reason to change the way it's been done until now.